blockchain/vm/contracts.go

```diff
@@ -78,7 +80,7 @@ var PrecompiledContractsConstantinople = map[common.Address]PrecompiledContract{
 	common.BytesToAddress([]byte{11}): &validateSender{},
 }
 
-// TODO-IstanbulCompatible: add blake2b and reprice bn_128 precompiled contract
+// DO NOT USE 0x3FD, 0x3FE, 0x3FF ADDRESSES BEFORE ISTANBUL CHANGE ACTIVATED.
```

Comment:
```suggestion
```
This comment seems inappropriate since `PrecompiledContractsIstanbul` is valid after istanbul change.

Reply:
@winnie-byun I add the line because the contracts deployed before Istanbul change will use `PrecompiledContractsConstantinople` due to precompiled contract address change item.